content/sensu-go/5.19/sensuctl/reference.md

```diff
@@ -648,6 +649,97 @@ See the [RBAC reference][21] for information about using access control with nam
 
 See the [RBAC reference][22] for information about local user management with sensuctl.
 
+## Prune resources
+
+{{% notice important %}}
+**IMPORTANT**: `sensuctl prune` is experimental and under testing for Sensu Go 5.19.0.
+We may remove this feature from 5.19.0 and future releases.
```

Comment:
I don't think we anticipate removing the feature (although I'm sure we could). I would try rephrasing this to frame that `sensuctl prune` is an alpha feature where breaking changes are still permitted.